#! /bin/sh /usr/share/dpatch/dpatch-run
## 21_remove_-v_option.dpatch by Jose Carlos Medeiros <debian@psabs.com.br>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove -v option.

######################################
# From: Jari Aalto <jari.aalto@cante.net>
# http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=308190
# Remove option -v (de facto verbose), and use only -V for version
#
# This patch remove "-v" option commonly interpreted as being same as --verbose.
# Now just anounce "-V"
# because -v is commonly known for being used for synonym of
# --verbose, like in
#
#    ssh -v -v -v ...


@DPATCH@
diff -urNad webalizer-2.01.10~/lang2.h webalizer-2.01.10/lang2.h
--- webalizer-2.01.10~/lang2.h	2006-07-11 20:04:55.000000000 -0300
+++ webalizer-2.01.10/lang2.h	2006-07-11 20:07:05.000000000 -0300
@@ -36,7 +36,7 @@
 /* Help display... */
 char *h_msg[]= {
          gettext_noop("-h        = print this help message")             ,
-         gettext_noop("-v -V     = print version information")           ,
+         gettext_noop("-V        = print version information")           ,
          gettext_noop("-d        = print additional debug info")         ,
          gettext_noop("-F type   = Log type.  type= (clf | ftp | squid)"),
          gettext_noop("-f        = Fold sequence errors")                ,
diff -urNad webalizer-2.01.10~/po/webalizer.pot webalizer-2.01.10/po/webalizer.pot
--- webalizer-2.01.10~/po/webalizer.pot	2006-07-11 20:04:55.000000000 -0300
+++ webalizer-2.01.10/po/webalizer.pot	2006-07-11 20:08:26.000000000 -0300
@@ -251,7 +251,7 @@
 msgstr ""

 #: lang2.h:39
-msgid "-v -V     = print version information"
+msgid "-V        = print version information"
 msgstr ""

 #: lang2.h:40
diff -urNad webalizer-2.01.10~/webalizer.1 webalizer-2.01.10/webalizer.1
--- webalizer-2.01.10~/webalizer.1	2006-07-11 20:04:55.000000000 -0300
+++ webalizer-2.01.10/webalizer.1	2006-07-11 20:07:16.000000000 -0300
@@ -167,7 +167,7 @@
 .B \-h
 Display all available command line options and exit program.
 .TP 8
-.B \-v \-V
+.B \-V
 Display program version and exit program.
 .TP 8
 .B \-d
diff -urNad webalizer-2.01.10~/webalizer.c webalizer-2.01.10/webalizer.c
--- webalizer-2.01.10~/webalizer.c	2006-07-11 20:04:56.000000000 -0300
+++ webalizer-2.01.10/webalizer.c	2006-07-11 20:06:50.000000000 -0300
@@ -360,7 +360,6 @@
         case 'T': time_me=1;                 break;  /* TimeMe              */
         case 'u': add_nlist(optarg,&hidden_urls);   break; /* hide URL      */
         case 'U': ntop_urls=atoi(optarg);    break;  /* Top urls            */
-        case 'v':
         case 'V': print_version(current_locale); break;  /* Version         */
 #ifdef USE_GEOIP
         case 'w': use_geoip=0;               break;  /* Disable GeoIP       */
